### PR TITLE
Initial version of nnp CI trigger script

### PR DESCRIPTION
**Pull request by abhiram-krishnan**
Adds the nnp CI trigger script which currently runs a few iterations of RN50 training as a CI check. Results are posted under build status ngtf-nnp.

### CODE REVIEW

File: .ci/jenkins/jenkins-trigger-nnp.groovy [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+JENKINS_DIR = '.'
+
+timestamps {
+    node("trigger") {
+
+        deleteDir()  // Clear the workspace before starting
+        // Clone the cje-algo directory which contains our Jenkins groovy scripts
+        try {
+            sh "git clone -b $JENKINS_BRANCH https://gitlab.devtools.intel.com/AIPG/AlgoVal/cje-algo.git ."
+        } catch (e) {
```
Comment (shresthamalik) [inline .ci/jenkins/jenkins-trigger-nnp.groovy]: Can we use the test/ci/jenkins folder to host this script?

File: .ci/jenkins/jenkins-trigger-nnp.groovy [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            echo "${e}"
+            println("ERROR: An error occurred during cje-algo script checkout.")
+            throw e
+        }
+
+        echo "Calling ngtf-bridge-ci-premerge.groovy"
+        def ngtfbrCIPreMerge = load("${JENKINS_DIR}/ngraph-tf-bridge-ci-premerge.groovy")
+        ngtfbrCIPreMerge(PR_URL, PR_COMMIT_AUTHOR, JENKINS_BRANCH, TIMEOUTTIME, PR_TARGET)
+        echo "ngtf-bridge-ci-premerge.groovy completed"
```
Comment (shresthamalik) [inline .ci/jenkins/jenkins-trigger-nnp.groovy]: We would want to add single chip RN50 run also. Do you want to do that in this PR or next?

Comment (abhiram-krishnan) [reply .ci/jenkins/jenkins-trigger-nnp.groovy]: Single chip is in my list of enhancements for the job - distributed build on Ubuntu, running bridge unit tests for NNP are also in the plan. Planning to add these as an enhancement once the initial version gets in. Let me know if that's okay with you.

File: .ci/jenkins/jenkins-trigger-nnp.groovy (hunk 1 above)
Comment (abhiram-krishnan) [reply .ci/jenkins/jenkins-trigger-nnp.groovy]: Done. Thanks for the suggestion. I wasn't aware the "test" folder was already used for storing buildkite scripts.

**Pull request merged**